### PR TITLE
add chart for redis evited keys

### PR DESCRIPTION
**Pull request by bbaronSVK**

### CODE REVIEW

File: python.d/redis.chart.py [hunk 1]
```diff
@@ -18,10 +18,15 @@
 #             'unix_socket': None
 #          }}
 
-ORDER = ['operations', 'hit_rate', 'memory', 'keys', 'net', 'connections', 'clients', 'slaves', 'persistence',
+ORDER = ['operations', 'hit_rate', 'memory', 'keys', 'net', 'connections', 'clients', 'eviction', 'slaves', 'persistence',
          'bgsave_now', 'bgsave_health']
 
 CHARTS = {
+    'eviction':{
+        'options': [None, 'Evicted keys', 'keys/s', 'evicted', 'redis.evictios', 'line'],
```
Comment (l2isbad) [inline python.d/redis.chart.py]: `Evicted Keys`
Are you sure about `absolute`? If so then `keys/s` is wrong should be `keys`.

**Comment by bbaronSVK** [comment]
You are right, i fix it.

File: python.d/redis.chart.py [hunk 2]
```diff
@@ -18,10 +18,15 @@
 #             'unix_socket': None
 #          }}
 
-ORDER = ['operations', 'hit_rate', 'memory', 'keys', 'net', 'connections', 'clients', 'slaves', 'persistence',
+ORDER = ['operations', 'hit_rate', 'memory', 'keys', 'net', 'connections', 'clients', 'eviction', 'slaves', 'persistence',
          'bgsave_now', 'bgsave_health']
 
 CHARTS = {
+    'eviction':{
+        'options': [None, 'Evicted keys', 'keys', 'evicted', 'redis.evictios', 'line'],
```
Comment (l2isbad) [inline python.d/redis.chart.py]: sorry for not being clear. `keys/s` is ok when algorithm is `incremental` and wrong if `absolute`. Netdata applies interpolation to find the exact value at second-boundaries only for `incremental` values, so `/s` is misleading for `absolute`.

Now units should be `keys/s` 😄

Comment (l2isbad) [reply python.d/redis.chart.py]: https://github.com/firehol/netdata/pull/1758#issuecomment-279153641

File: python.d/redis.chart.py [hunk 3]
```diff
@@ -18,10 +18,15 @@
 #             'unix_socket': None
 #          }}
 
-ORDER = ['operations', 'hit_rate', 'memory', 'keys', 'net', 'connections', 'clients', 'slaves', 'persistence',
+ORDER = ['operations', 'hit_rate', 'memory', 'keys', 'net', 'connections', 'clients', 'eviction', 'slaves', 'persistence',
          'bgsave_now', 'bgsave_health']
 
 CHARTS = {
+    'eviction':{
+        'options': [None, 'Evicted keys', 'keys', 'evicted', 'redis.evictios', 'area'],
```
Comment (l2isbad) [inline python.d/redis.chart.py]: > redis.evictios

typo

**Comment by ktsaou** [comment]
merged it

**Pull request closed without merging**